### PR TITLE
Updated to work with NXRM 3.30.0-01

### LINKED ISSUE #33: Update to work with NXRM 3.30.0-01
Update to work with NXRM 3.30.0-01.

I will submit a PR with the changes

cc @DarthHater @bhamail

### PR DESCRIPTION
**Pull request by mikeq**
Updated to work with NXRM 3.30.0-01

This pull request makes the following changes:
* Update version numbers in the pom files

I am not a Java developer so I have made the changes I thought were required to make the plugin work with NXRM 3.30.0-01 and tested it works with that version of NXRM. If there is something else that needs to be done just let me know.

* Fixes #33 

cc @DarthHater @bhamail

### CODE REVIEW

**Comment by joedragons** [comment]
Looks fine to me tho not sure if the change to -19 is needed or done by another process.  Defer to others you CCed.  Thanks for the contribution.

File: nexus-repository-apk-it/pom.xml [hunk 1]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>apk-parent</artifactId>
-    <version>0.0.18-SNAPSHOT</version>
+    <version>0.0.19-SNAPSHOT</version>
```
Comment (bhamail) [inline nexus-repository-apk-it/pom.xml:8]: You should be able to revert this change, and the "maven release" process takes care of releasing version `0.0.18` and incrementing the development version to the next revision (`0.0.19-SNAPSHOT`).

File: nexus-repository-apk/pom.xml [hunk 2]
```diff
@@ -5,7 +5,7 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>apk-parent</artifactId>
-    <version>0.0.18-SNAPSHOT</version>
+    <version>0.0.19-SNAPSHOT</version>
```
Comment (bhamail) [inline nexus-repository-apk/pom.xml:8]: see prior comment. can just revert this change.

File: pom.xml [hunk 3]
```diff
@@ -15,11 +15,11 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.29.2-02</version>
+    <version>3.30.0-01</version>
   </parent>
 
   <artifactId>apk-parent</artifactId>
-  <version>0.0.18-SNAPSHOT</version>
+  <version>0.0.19-SNAPSHOT</version>
```
Comment (bhamail) [inline pom.xml:22]: see prior comment. can just revert this change.

**Review by bhamail — CHANGES_REQUESTED** [review]
Looks good! Please revert the version bump as noted in my comments.
I'm also looking into why the CI build didn't appear to run (unrelated to your changes).

**Comment by bhamail** [comment]
One more minor request: Could you update the README.md file, adding to the `Plugin Version` table:

| v0.0.12        | 3.28.0-01                |

becomes:

| v0.0.12        | 3.28.0-01                |
| v0.0.18        | 3.30.0-01                |

**Comment by bhamail** [comment]
Thanks @mikeq ! Merging now. (Still now certain why the CI didn't fire, but I verified it works with a local CI build).

**Pull request merged**